plugin/src/main/java/org/wso2/carbon/uuf/maven/bean/AppConfig.java

```diff
@@ -31,10 +32,12 @@
 public class AppConfig {
 
     private static final Pattern HTTP_STATUS_CODES_PATTERN = Pattern.compile("[1-5][0-9][0-9]");
+    private static final Pattern JAVA_CLASS_NAME_PART_PATTERN = Pattern.compile("[A-Za-z_$]+[a-zA-Z0-9_$]*");
```

Comment:
Instead of this pattern, use java.lang.Character classes `p{javaJavaIdentifierPart}` and `\p{javaJ‌​avaIdentifierPart}` in this pattern. Refer [this SO answer](http://stackoverflow.com/a/30661779/1577286) for more info.